decidim-forms/app/presenters/decidim/forms/admin/questionnaire_answer_presenter.rb

```diff
@@ -52,11 +55,21 @@ def pretty_attachment(attachment)
           # rubocop:enable Style/StringConcatenation
         end
 
-        def choice(choice_body)
-          content_tag :li do
-            choice_body
+        def choice(choice_hash)
+          render_body_for(choice_hash[:answer_option_body], :em) + render_body_for(choice_hash[:choice_body])
+        end
+
+        def render_body_for(body, content_tag = :li)
+          content_tag content_tag do
+            body
           end
         end
+
+        def body_or_custom_body(choice)
+          return choice.custom_body if choice.try(:custom_body).present?
+
+          choice.try(:body).present? ? "-" : ""
```

Comment:
I don't like this "-" thing, but as far as I see this is already the behavior of `decidim-forms`, so we'll leave it as is